Pod/Classes/Operations/SKYRegisterDeviceOperation.h

```diff
@@ -22,7 +22,9 @@
 
 @interface SKYRegisterDeviceOperation : SKYOperation
 
-- (instancetype)initWithDeviceToken:(NSData *)deviceToken;
+- (instancetype)initWithDeviceToken:(NSData *)deviceToken
+    __attribute__((deprecated("Use -(void)initWithDeviceToken:topic: instead")));
```

Comment:
interesting, i didn’t know about this
no need for the return type of the method here, and btw the return type of the method is incorrect.

Ditto for below.